packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -438,15 +469,14 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
 
   /**
    * @notice Computes a tobin tax based on the reserve ratio.
-   * @return The numerator of the tobin tax amount, where the denominator is 1000.
+   * @return The tobin tax expresesed as a fixidity fraction.
    */
   function computeTobinTax() private view returns (FixidityLib.Fraction memory) {
-    // The protocol calls for a 0.5% transfer tax on Celo Gold when the reserve ratio <= 2.
     FixidityLib.Fraction memory ratio = FixidityLib.wrap(getReserveRatio());
-    if (ratio.gte(FixidityLib.newFixed(2))) {
+    if (ratio.gte(FixidityLib.wrap(tobinTaxReserveRatio))) {
       return FixidityLib.wrap(0);
     } else {
-      return FixidityLib.wrap(TOBIN_TAX_NUMERATOR);
+      return FixidityLib.wrap(tobinTax);
```

Comment:
ditto with `tobinTax`? we would save gas by computing this in the setters

Reply:
in fact we do compute it in the setter but don't store the value

Reply:
(mixed up `wrap` and `newFixed` but we would still save some albeit minimal amt of gas

Reply:
See comment above.